docs/frontend/page-layout-design.md

```diff
@@ -0,0 +1,62 @@
+# Page layout design
+Page layout is divided to three main parts
+- web__header
```

Comment:
hmm if this is talking about classes, shouldn't it be with a dot? `.web__header` (and the other classes as well)?

Reply:
Thx for your note. Yes it will be better to use dot before class names. Fixed

Reply:
I added updgrade notes for this changes.